Buổi 10/homework.js

```diff
@@ -0,0 +1,146 @@
+//Bài 1
+const Multi = number =>{
+    for(i=1;i<11;i++){
+        console.log(`${number} x ${i} = ${number*i}`)
+    }
+}
+
+
+//Bài 2
+const EvenNumber = number =>{
+    if(number<=30){
+        for(i=1;i<=number;i++){
+            if(i%2==0){
+                console.log(i)
+            }
+        }
+    }
+    else{
+        return false
+    }
+}
+
+//Bài 3
+const Sum = number =>{
+    let sum=0
+    if(number>0 && number<=30){
+        for(i=1;i<=number;i++){
+            sum+=i
+        }
+        return sum
+    }
+    else{
+        return false
+    }
+}
+
+//Bài 4
+const Factorial = number =>{
+    let factorial = 1
+    if(number<=30 && number>0){
+        for(i=1;i<=number;i++){
+            factorial*=i
+        }
+        return factorial
+    }
+    else{
+        return false
+    }
+}
+
+//Bài 5
+const EvenCount = (arr=[])=>{
+    let count=0
+    for(i=0;i<arr.length;i++){
+        if(arr[i]%2==0){
+            count++
+        }
+    }
+    return count
+}
+
+//Bài 6
+const Remove = (arr=[])=>{
+    var newarr=[]
+    arr.forEach(function(item,index){
+        if(arr.indexOf(item)===index){
+            newarr.push(item)
+        }
+    })
+    return newarr
+}
+
+
+//Bài 7
+const studentNames = [
+    { id: 1, name: 'Nguyễn Thị Tèo' },
+    { id: 2, name: 'Phạm Văn Bưởi' },
+    { id: 3, name: 'Hoàng Văn Nam' },
+    { id: 4, name: 'Vũ Ngọc Duy' },
+    { id: 5, name: 'Nguyễn Minh Nhật' },
+    { id: 6, name: 'Phí Duy Quân' },
+    { id: 7, name: 'Trần Minh Minh' }
+]
+const studentScores = [
+    { id: 1, score: 9.2 },
+    { id: 2, score: 2.3 },
+    { id: 3, score: 3.7 },
+    { id: 4, score: 6.9 },
+    { id: 5, score: 5.2 },
+    { id: 6, score: 9.6 },
+    { id: 7, score: 6.1 }
+]
+
+const Connect = (studentNames,studentScores)=>{

```

Comment:
1. Tên hàm phải là động từ, phải mô tả được nội dung hàm đang làm gì
2. Các params cách nhau bằng dấu `,` sau dấu `,` thì thêm 1 dấu cách
```suggestion
const Connect = (studentNames, studentScores) => {

```